WooCommerce/Classes/Analytics/WooAnalyticsEvent+Themes.swift

```diff
@@ -0,0 +1,74 @@
+import Foundation
+
+extension WooAnalyticsEvent {
+
+    enum Themes {
+        private enum Key: String {
+            case source
+            case theme
+            case layout
+            case page
+            case pageURL = "page_url"
+        }
+
+        enum Source: String {
+            case storeCreation = "store_creation"
+            case settings
+        }
+
+        /// Tracked when the theme picker screen is displayed
+        static func pickerScreenDisplayed(source: Source) -> WooAnalyticsEvent {
+            .init(statName: .themePickerScreenDisplayed, properties: [
+                Key.source.rawValue: source.rawValue
+            ])
+        }
+
+        /// Tracked when a theme screenshot is tapped on the theme picker screen
+        static func themeSelected(id: String) -> WooAnalyticsEvent {
+            .init(statName: .themePickerThemeSelected, properties: [
+                Key.theme.rawValue: id
+            ])
+        }
+
+        /// Tracked when the theme preview screen is displayed
+        static func previewScreenDisplayed() -> WooAnalyticsEvent {
+            .init(statName: .themePreviewScreenDisplayed, properties: [:])
+        }
+
+        /// Tracked when the user selected a layout to be previewed
+        static func previewLayoutSelected(layout: ThemesPreviewView.PreviewDevice) -> WooAnalyticsEvent {
+            .init(statName: .themePreviewLayoutSelected, properties: [
+                Key.layout.rawValue: layout.rawValue
+            ])
+        }
+
+        /// Tracked when the user selected a page to be previewed
+        static func previewPageSelected(page: String, url: String) -> WooAnalyticsEvent {
+            .init(statName: .themePreviewPageSelected, properties: [
+                Key.page.rawValue: page,
+                Key.pageURL.rawValue: url
+            ])
+        }
+
+        /// Tracked when the “Start with the theme” button on preview screen is tapped
+        static func startWithThemeButtonTapped(themeID: String) -> WooAnalyticsEvent {
+            .init(statName: .themePreviewStartWithThemeButtonTapped, properties: [
+                Key.theme.rawValue: themeID
```

Comment:
1. Can we update the PT that we are tracking themeID instead of name? 
1. Can the key be kept as `theme` or should we change it as `themeID`? 
<img width="744" alt="image" src="https://github.com/woocommerce/woocommerce-ios/assets/524475/af518f13-b34f-4510-8ff4-475858e2716e">

Reply:
> Can we update the PT that we are tracking themeID instead of name?

Done ✅  Thanks for the reminder!

> Can the key be kept as theme or should we change it as themeID

I think `theme` should still be sufficient, it's also simpler. Let's keep it that way.

I also updated the source from `profiler` to `store_creation` following p1702963419256829/1702955464.357419-slack-C03L1NF1EA3.